lesson03/assignment/scripts/script.js

```diff
@@ -0,0 +1,86 @@
+/* linkedIn_data array represents local JSON data */
+const linkedIn_data = [{
+    "name": "Steve Smith",
+    "jobTitle": "Project Manager",
+    "company": "Front End Dev Co",
+    "experience": "3 years",
+    "school": "UW",
+    "major": "Marketing",
+    "email": "steve@fedc.com",
+    "linkedInUrl": "steve.linkedinprofile.com",
+    "codeLanguages": [
+      "HTML", "CSS", "JavaScript", ".NET", "C#"
+    ]
+  },/*something weird is occurring with second object*/
+  {
+    "name": "Aaron Katz",
+    "jobTitle": "Full Stack Developer",
+    "company": "Web Sites and More",
+    "experience": "5 years",
+    "school": "SU",
+    "major": "Computer Science",
+    "email": "aaronNotMyemail@uw.com",
+    "linkedInUrl": "aaron.linkedinprofile.com",
+    "codeLanguages": [
+      "HTML", "CSS", "JavaScript", "PHP", "Twig"
+    ]
+  },
+  {
+    "name": "Kyle Hendricks",
+    "jobTitle": "Starting Pitcher",
+    "company": "Chicago Cubs",
+    "experience": "12 years",
+    "school": "USC",
+    "major": "Pitching",
+    "email": "kyleH@cubs.com",
+    "linkedInUrl": "kyle.linkedinprofile.com",
+    "codeLanguages": [
+      "HTML", "CSS", "JavaScript", "Ruby", "Rails"
+    ]
+  },
+  {
+    "name": "Michael Jordan",
+    "jobTitle": "Point Guard",
+    "company": "Chicago Bulls",
+    "experience": "20 years",
+    "school": "UNC",
+    "major": "Trash Talking",
+    "email": "mJordan@bulls.com",
+    "linkedInUrl": "mJordas.linkedinprofile.com",
+    "codeLanguages": [
+      "HTML", "CSS", "JavaScript", "Java", "Spring"
+    ]
+  }
+];
+
+/*creating template function*/
+const linkedInHTML = linkedIn_data.map(function(template) {
```

Comment:
your JSON data is constructed well, and you're mapping the data correctly. Nothing to add here!